api/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "strict": true,
```

Comment:
we can remove `noImplicityAny` and `strictNullChecks`

Reply:
Fixed in https://github.com/Vizzuality/marxan-cloud/pull/135/commits/d8ef02663ae149b88043f29404e4f763f1ad337e